test/integration/take_part_page_test.rb

```diff
@@ -70,4 +70,107 @@ class TakePartPageTest < ActiveSupport::TestCase
       )
     end
   end
+
+  test "TakePartPage patches links in the correct order in the Get Involved Page when created" do
+    Sidekiq::Testing.inline! do
+      # Working with a specific ID
+      get_involved_content_id = "dbe329f1-359c-43f7-8944-580d4742aa91"
```

Comment:
I'd be tempted to move this into a shared variable in the `setup` block, i.e. `@get_involved_content_id`, to remove the duplication of this ID throughout the file.